ssm_cache/cache.py

```diff
@@ -96,9 +71,79 @@ def wrapped(*args, **kwargs):
                     return func(*args, **kwargs)
                 except error_class:
                     self.refresh()
-                    if callable(error_callback):
+                    if error_callback:
                         error_callback()
-                    kwargs[retry_argument] = True
+                    if retry_argument:
+                        kwargs[retry_argument] = True
                     return func(*args, **kwargs)
             return wrapped
         return true_decorator
+
+class SSMParameterGroup(Refreshable):
+    def __init__(self, max_age=None, with_decryption=True):
+        super(SSMParameterGroup, self).__init__(max_age)
+        
+        self._with_decryption = with_decryption
+        self._parameters = {}
+    
+    def parameter(self, *args, **kwargs):
```

Comment:
what happens if I run the following?

```
param1 = group.parameter("my_name")
same_name_param = group.parameter("my_name")
```

Should we raise or return the already-existing `SSMParameter` object?

Reply:
Good question! I think we should return the exist object. I also realized this means we can't let with_decryption be set individually on grouped parameters either.